### ISSUE TITLE
Error when using server.stop() and good-file

### ISSUE THREAD

**Issue by pierre-b** [hapijs/good-file#73]
Hello guys, here is the error when I call server.stop() having registered good-file plugin:
```
/my_project/node_modules/good-file/node_modules/big-time/lib/index.js:64
  return timer.close();
              ^

TypeError: Cannot read property 'close' of null
    at Object._clearTimeout [as clearTimeout] (/my_project/node_modules/good-file/node_modules/big-time/lib/index.js:64:15)
    at null.<anonymous> (/my_project/node_modules/good-file/lib/index.js:146:12)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at internals.Monitor.stop (/my_project/node_modules/good/lib/monitor.js:220:10)
    at null.<anonymous> (/my_project/node_modules/good/lib/index.js:14:17)
    at emitNone (events.js:67:13)
    at emit (events.js:166:7)
    at EventEmitter.handler (/my_project/node_modules/hapi/node_modules/kilt/lib/index.js:56:44)
    at emitNone (events.js:67:13)
```
You can test it yourself with:
```javascript
var Hapi = require('hapi'),
    serverStopped = false;

var server = new Hapi.Server({ debug: { request: ['error'] } });

server.connection({ port: 1234 });

var stopGracefully = function(code){

  if(serverStopped) return;
  
  console.log('stopping server gracefully...');

  serverStopped = true;

  server.stop({timeout: 30000}, function(){
    console.log('all connections closed.');
    process.exit(code);
  });
};


// HANDLE KILL EVENTS

process.on("uncaughtException", function(err) {
  console.error(err);
  return stopGracefully(1); // 1 : failure
});
 
process.on("SIGTERM", function() {
  return stopGracefully(0);
});
 
process.on("SIGINT", function() {
  return stopGracefully(0);
});


server.register({
  register: require('good'),
  options: {
    reporters: [{
      reporter: require('good-console'),
      events: {
        response: '*',
        log: '*',
        error: '*' 
      }
    },
    {
      reporter: require('good-file'),
      config: './api.log',
      events: {
        log: '*',
        error: '*'
      }
    }]
  }
}, function (error) { 
  if(error) console.error(error);

  server.start(function () {
    server.log('info', 'Server running');

    setTimeout(function(){
      stopGracefully();
    }, 3000)
  });
});
```

Any idea please?

**Comment by arb** [comment]
Thanks for finding this. Published version 5.1.1 with a fix.

**Comment by pierre-b** [comment]
Thanks for the quick fix ;)
